service/dap/server_test.go

```diff
@@ -364,11 +364,11 @@ func TestAttachStopOnEntry(t *testing.T) {
 		// 13 >> disconnect, << disconnect
 		client.DisconnectRequestWithKillOption(true)
 
-		// Both of these scenarios are somehow possible.
```

Comment:
You removed the comment, but not the code for the alternative case, which shouldn't happen anymore.  I think in this case, it should go back to the original logic that this commit expanded.
https://github.com/go-delve/delve/pull/2423/commits/9c8e957134b1e271bb728f787712803831848385

Reply:
As far as I can see that wouldn't work because the stopped event can be received at any point (before the output, between the output and the disconnect or after the disconnect).

Reply:
Right, but there shouldn't be a TerminatedEvent in the mix anymore, which is gone now.

Reply:
I believe I have now removed that check.